process-engine/process_engine_runtime
# ♻️ Refactor base flow node handlers

**Changes:**

1. Change the schema type for `FlowNodeInstance.error` to `TEXT`
    - Using `STRING` here is insufficient, because it would limit the column to 255 chars with postgres. This is not enough for error objects.
2. Add migration script to cover this change.
3. Apply refactoring of the base FlowNodeHandlers from the ProcessEngineCore package.

**Issues:**

Closes #314 

PR: #315

## How can others test the changes?

Testing the refactored error column:
- Run a process and provoke an error on a FlowNode, using Postgres as database
- See that the error gets stored correctly.

The refactoring of the base FlowNodeHandlers shouldn't be visible to outside users at all.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
